bitcoin-rpc-provider/src/lib.rs

```diff
@@ -172,6 +166,15 @@ fn enc_err_to_manager_err(_e: EncodeError) -> ManagerError {
     Error::BitcoinError.into()
 }
 
+fn u8slice_to_string(input: &[u8]) -> String {
+    use core::fmt::Write;
+    let mut ret = String::with_capacity(2 * input.len());
+    for ch in input {
+        write!(ret, "{:02x}", ch).expect("writing to string");
+    }
+    ret
+}
+
```

Comment:
all the `rust-bitcoin` projects are now using the `hex-conservative` crate, could be worth just using that here

Reply:
Done